CHANGELOG.md

```diff
@@ -7,13 +7,30 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 ### Breaking Changes
 
+- Message format was changed, that modify byte representation of
+   transaction and precommit. (#916)
+
+- `Transaction::execute` now accept as second parameter `TransactionContext`
+   which provide public key of transaction author, id of current service,
+   and transaction hash. (#943)
+
+- Every transaction that contain public key of author was refactored to
+   author from `TransactionContext`. (#984 #980 #979 #975 #971)
```

Comment:
refactored to author from `TransactionContext` - perhaps it could be rephrased somehow. Maybe to something like the following, is it is technically correct: refactored to be assigned to the author indicated in `TransactionContext`.